components/ApplicationNameBar.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react'
+
+import Link from 'next/link'
+
+export interface ApplicationNameBarProps {
+  text: string
+  href: string
+}
+
+const ApplicationNameBar: FC<ApplicationNameBarProps> = ({ text, href }) => {
+  return (
+    <div id="app-bar" className="bg-blue-dark">
+      <section className="container mx-auto p-4">
+        <Link href={href} passHref>
```

Comment:
You don't need `passHref`. Use `passHref` if the child is a custom component that wraps an <a> tag.